api.docs/includes/_organizations.md

```diff
@@ -262,14 +266,70 @@ mutation createOrganization($input:OrganizationInput!) {
 
 ### Query Parameters
 
-Parameter | Type | Default | Description
---------- | ---- | ------- | -----------
-input | <a href="#organizationinput">OrganizationInput</a> | required ||
+| Parameter | Type                                               | Default  | Description |
+| --------- | -------------------------------------------------- | -------- | ----------- |
+| input     | <a href="#organizationinput">OrganizationInput</a> | required |             |
+
+### Return Parameters
+
+| Type                                                 | Description                     |
+| ---------------------------------------------------- | ------------------------------- |
+| <a href="#organizationresult">OrganizationResult</a> | The created organization object |
+
+## Update a Organization Fields
```

Comment:
WHy do we need this, cant we just reuse updateOrganization as the API, with fields being a parameter?